Azure/sonic-buildimage
# [Celestica] Fix Seastone2 LED init script to not break port status

#### Why I did it
Fixes #10103.

The LED init script that was added in #9769 enables the port LEDs, but it turns out that the linkscan change combined with Broadcom SAI 6.0 breaks detection of port status on boot.
#### How I did it
Without the linkscan and sleep, the LEDs still work as expected, and the port status is correct again.

Fixed script was provided by @jonwhaley: https://github.com/Azure/sonic-buildimage/issues/10103#issuecomment-1120047125
#### How to verify it
Apply the change on a Seastone2 switch, reboot, observe that port LEDs work and link status is detected.

#### Which release branch to backport (provide reason below if selected)


#9769 is only included in 202111.

#### Description for the changelog
Fix Seastone2 port status detection on boot.

#### Link to config_db schema for YANG module changes
#### A picture of a cute animal (not mandatory but encouraged)
![image (6)](https://user-images.githubusercontent.com/10601319/168306011-8e470b2b-ff57-4405-bde5-581cb313ec82.png)
